cmd/weak-key-search/main_test.go

```diff
@@ -0,0 +1,76 @@
+	test.AssertNotError(t, err, "hex.DecodeString failed")
+	wk, err := x509.ParsePKCS1PrivateKey(wkBytes)
+	test.AssertNotError(t, err, "x509.ParsePKCS1PrivateKey failed")
+	derB, err := x509.CreateCertificate(rand.Reader, &temp, &temp, &wk.PublicKey, wk)
+	test.AssertNotError(t, err, "x509.CreateCertificate failed")
+
+	now := time.Now()
+	err = dbMap.Insert(
+		&core.Certificate{RegistrationID: reg.ID, Serial: "cert-a", DER: derA, Issued: now.Add(time.Hour * -2)},
+		&core.Certificate{RegistrationID: reg.ID, Serial: "cert-b", DER: derA, Issued: now},
+		&core.Certificate{RegistrationID: reg.ID, Serial: "cert-c", DER: derB, Issued: now},
+		&core.Certificate{RegistrationID: reg.ID, Serial: "cert-d", DER: derA, Issued: now.Add(time.Hour * 2)},
+	)
+	test.AssertNotError(t, err, "dbMap.Insert failed")
+
+	limit = 1
+	w := make(chan certInfo, 10)
+	getCerts(w, now.Add(-time.Hour), now.Add(time.Hour), dbMap.Db)
+	test.AssertEquals(t, len(w), 2)
+
+	tempDir, err := ioutil.TempDir("", "weak-keys")
+	test.AssertNotError(t, err, "Failed to create temporary directory")
+	tempPath := filepath.Join(tempDir, "a.json")
+	err = ioutil.WriteFile(tempPath, []byte("[\"8df20e6961a16398b85a\"]"), os.ModePerm)
```

Comment:
It might be worth a quick comment on this hex prefix and the `wkBytes` to describe where they came from. I guess it's fairly intuitive? <shrug>